src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
```

Comment:
Recommend adding constants for health and bite values ​​to avoid magic numbers in the code.
Example: 
```
const MAX_HEALTH = 100;
const BITE_DAMAGE = 50;
const DEAD_VALUE = 0;
```